.github/tests.yaml

```diff
@@ -0,0 +1,31 @@
+betelgeuse: "tests/foreman/ -v -m 'build_sanity'"
+broker[docker]: "tests/foreman/ -v -m 'build_sanity'"
+cryptography: "tests/foreman/ -v -m 'build_sanity'"
```

Comment:
I would rather run tests playing with cryptography or even we can run unittests for cryptography.

Reply:
Can you please point me out how we can achieve running these unit tests?

Reply:
As discussed yday, very same as running `tests/foreman` tests , run the `tests/robottelo` tests .

Reply:
these are already running part of pull_request.yml as
```
- name: Run Robottelo's Tests
  run: pytest -sv tests/robottelo/  
``` 

no need to run them again.